protocol.go

```diff
@@ -68,7 +68,7 @@ func (p *Listener) Accept() (net.Conn, error) {
 
 	proxyHeaderPolicy := USE
 	if p.Policy != nil {
-		proxyHeaderPolicy, err = p.Policy(conn.RemoteAddr())
+		proxyHeaderPolicy, err = p.Policy(conn.RemoteAddr(), conn.LocalAddr())
```

Comment:
I see now what the problem is. Let me think a little bit more about this. We'll find agreement.

Reply:
sure 👍

Reply:
@pires did you get a chance to look more into this?